app/memory_order.py

```diff
@@ -32,5 +26,8 @@ def _str_to_endian(self, order_string: str):
             case "BADC":
                 word_order = Endian.Big
                 byte_order = Endian.Little
+            case _:
```

Comment:
is this safe?

Reply:
I wonder if we should have an exception instead

Reply:
yes that's a better idea

Reply:
I feel is unsafe to assume things, specially if we have a process that ensures the config file has been filled correctly